packages/block-editor/src/utils/dom.js

```diff
@@ -96,7 +96,7 @@ function isElementVisible( element ) {
 		return false;
 	}
 
-	return element.checkVisibility( {
+	return element.checkVisibility?.( {
```

Comment:
Thank you for the quick fix @andrewserong!!


Should have some fallback for Safari 17.3? E.g., 

Warning - just an example: 😄 

```js
function isElementVisible(element) {
  if ( typeof element.checkVisibility === 'function' ) {
    return element.checkVisibility( {
      checkOpacity: true,
      checkVisibilityCSS: true
    } ;
  } else {
    const style = window.getComputedStyle( element) ;

    return style.display !== 'none' &&
           style.visibility !== 'hidden' &&
           style.opacity !== '0' &&
           element.offsetWidth > 0 &&
           element.offsetHeight > 0;
  }
}
```

Also, if we're going to the effort to support previous versions of Safari, should we update the [config](https://github.com/WordPress/gutenberg/blob/trunk/packages/browserslist-config/index.j) as well to formalize it?

Reply:
> Also, if we're going to the effort to support previous versions of Safari, should we update the [config](https://github.com/WordPress/gutenberg/blob/trunk/packages/browserslist-config/index.j) as well to formalize it?

Good idea. I'd think of that as a different PR though, so that it's a little more visible outside of a bug fix PR.

Reply:
> Should have some fallback for Safari 17.3?

Good call! I just used a truthy check instead of checking that it's a function in https://github.com/WordPress/gutenberg/pull/65069/commits/c94339954daae415866417b53fb153fdfb5c551d — happy to change that in any way. All nitpicks welcome!